server/views/pages/bookings/future.njk

```diff
@@ -25,7 +25,8 @@
             Booking reference: <span data-test="visit-reference-{{ loop.index }}">{{ visit.reference }}</span>
           </p>
           <p>
-            <a href="{{ paths.BOOKINGS.VISIT }}/{{ visit.visitDisplayId }}" class="govuk-link--no-visited-state" data-test="visit-link-{{ loop.index }}">View booking details</a>
+            <a href="{{ paths.BOOKINGS.VISIT }}/{{ visit.visitDisplayId }}" class="govuk-link--no-visited-state" data-test="visit-link-booking-{{ loop.index }}">View booking details</a>
+            <a href="{{ paths.BOOKINGS.CANCEL_VISIT }}/{{ visit.visitDisplayId }}" class="govuk-link--no-visited-state govuk-!-padding-left-3" data-test="visit-link-cancel-{{ loop.index }}">Cancel booking</a>
```

Comment:
Figma design has some hidden content for accessibility that should be incorporated into these links. (Also, text in the design is now `"View details"`, not `"View booking details"`.)